### DISCUSSION TITLE
Device interview stuck on "ProtocolInfo" for days

### DISCUSSION THREAD

**Discussion by lazloman** [zwave-js/zwave-js-ui#1088] [Q&A]
I removed garage controller which was node 7, but zwavejs does not recognize the change. There is a status of asleep, last active is never and the interview stays at ProtocolInfo. The gdc has since been re-included to the network and uses a different node id. I'm not sure its related, but all my other devices, except the controller itself, Nortek HUSBZB-1, all show up as "Unknown Manufacturer" and "Unknown Device". 
<img width="1440" alt="Screen Shot 2021-04-20 at 6 55 08 PM" src="https://user-images.githubusercontent.com/2359182/115477646-fe501a80-a209-11eb-9eac-776350801733.png">

**Comment by robertsLando** [comment]
@lazloman please provide a zwavejs log file

**Reply by robert6519** [reply]
I have the same problem:
![image](https://user-images.githubusercontent.com/85799168/132257155-70d5fae3-0e56-4673-9cf6-288edf0b465e.png)
![image](https://user-images.githubusercontent.com/85799168/132257267-57bd8b82-8ba8-434d-ba71-2ac349568e00.png)

**Reply by robert6519** [reply]
![image](https://user-images.githubusercontent.com/85799168/132257356-5ac3e81e-4af2-4a7c-b5e7-4bd7ce6096e2.png)

**Reply by robertsLando** [reply]
Did you try to manually wake up it?

**Comment by robert6519** [comment]
yes, i try everthing , press Buttons on the batterydevice, pull out the battery and back again.
manuel health device and so on.

I think there is a process still waiting on some messages from the device.
Also a reboot of the rasperry will not help.
It is alway the same state.

**Reply by robertsLando** [reply]
@AlCalzone

**Reply by robert6519** [reply]
Many thanks for the help, after some more trys the device doesn`t send anymore.
With a new battery the device is working again.

**Comment by AlCalzone** [comment]
Please make a [driver log](https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/bug_report?id=driver-logs), loglevel `debug` or `silly` and attach it here as a file.

While the log is being created, wake up the device in question.

---

> I removed garage controller which was node 7, but zwavejs does not recognize the change

Seems like you didn't exclude it properly. You can get rid of the device by using the "remove failed node" function.
